bin/get_import_data.py

```diff
@@ -1,281 +0,0 @@
-#!/usr/bin/env python
-
-"""
-    author Brian O'Connor
-    broconno@ucsc.edu
-    This module first crawsl the filesystem looking for manifest.json files, parses
-    them, finds data to download, and downloads them.
-    Example: python bin/get_import_data.py --input-dir import --output-s3-dir s3://hca-dss-test-src/data-bundle-examples --cleanup --test
-    For where Clay was working (note plural "bundles"):
-    git pull && python bin/get_import_data.py --input-dir import --output-s3-dir s3://hca-dss-test-src/data-bundles-examples --cleanup > >(tee stdout.txt) 2> >(tee stderr.txt >&2)
-    # an example that makes bundles in the blue box in addition to staging data 
-     git pull && time python bin/get_import_data.py --input-dir import --output-s3-dir s3://hca-dss-test-src/data-bundles-examples --cleanup --make-bundles > >(tee stdout.4.txt) 2> >(tee stderr.4.txt >&2)
-    Tested with Python 3.6.0
-"""
-
-import os
-import os.path
-import platform
-import argparse
-import json
-import jsonschema
-import datetime
-import re
-import dateutil
-import ssl
-import dateutil.parser
-import ast
-import sys
-import boto
-from boto.s3.key import Key
-from urllib.request import urlopen, urlretrieve
```

Comment:
For future reference, suggest using urllib3 which is pretty much better at http across the board than urllib.